sdk/core/core-lro/test/utils/testOperation.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+/* eslint-disable no-invalid-this */
```

Comment:
can't we provide a `this` typing to the function instead of disabling this check?

Reply:
I don't think that's what it's complaining about. It doesn't like the usage of `this` in anything it doesn't consider a class-like context according to some patterns:

https://eslint.org/docs/rules/no-invalid-this

Reply:
Yeah, no-invalid-this causes problems with many of our clients, including all of the KeyVault ones :/

Reply:
I created this issue https://github.com/Azure/azure-sdk-for-js/issues/11404.